javascript/javascript1/week1/FirstHomeWork.js

```diff
@@ -0,0 +1,113 @@
+// Age-ify (A future age calculator)
+console.log('---Age-ify---')
+const yearOfBirth = 1985
+const yearFuture = 2027
+
+const age = yearFuture - yearOfBirth
+
+console.log('How old will you be in ' + yearFuture + '?')
+console.log('You will be ' + age + ' years old in ' + yearFuture)
+
+// Goodboy-Oldboy (A dog age calculator)
+console.log('---Goodboy-Oldboy---')
+
+// I didn’t quite understand how dog years lead to human years. But according to the assignment, it is clear that 1 to 7 did so
+
+const dogYearOfBirth = 2015
+const dogYearFuture = 2027
+const dogYear = dogYearFuture - dogYearOfBirth
+
+var shouldShowResultInDogYears = new Boolean(false)
```

Comment:
As we discussed, we should use `const` or `let` instead of var :).
Also `false` instead of `new Boolean(false)` is acceptable